File: despatch.WEB/app/src/swagger/models/contact-data-urls-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { ContactDataRequest } from './contact-data-request';
import { UrlRequest } from './url-request';
export interface ContactDataUrlsRequest extends BaseEntityRequest{
  contactData?: ContactDataRequest;
  contactDataId?: string;
  url?: UrlRequest;
  urlId?: string;
}

File: despatch.Domain/Models/Default/Request/ContactDataUrlsRequest.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace despatch.Domain.Models.Default.Request
{
    public class ContactDataUrlsRequest : BaseEntityRequest
    {
        public string UrlId { get; set; }
        public string ContactDataId { get; set; }

        public virtual UrlRequest Url { get; set; }
        public virtual ContactDataRequest ContactData { get; set; }
    }
}
